src/js/controller/controller.js

```diff
@@ -165,10 +165,12 @@ Object.assign(Controller.prototype, {
                 model.setStreamType(type);
             });
 
-            const recsAuto = (model.get('related') || {}).oncomplete === 'autoplay';
             const index = model.get('item') + 1;
+            const recsAuto = (model.get('related') || {}).oncomplete === 'autoplay';
             let item = model.get('playlist')[index];
-            if ((item || recsAuto) && Features.backgroundLoading) {
+            // Do not background load DAI items because that item will be dynamically replaced
+            const isValidItem = (item && !item.daiSetting) || recsAuto;
```

Comment:
Since `item` can be set on position, this should be done in the event handler, in the event that "nextUp" has `daiSetting`.

Reply:
I assumed recs are not configured to play with DAI

Reply:
They are not in general, but there is no reason why they could not be.